README.md

```diff
@@ -81,23 +81,30 @@ outside this list)
         signing.gnupg.passphrase=${password_for_imported_sumoapi_key}
         ```
 2. Remove `-SNAPSHOT` suffix from `version` in `build.gradle`
-3. Make a release branch with Scala version and project version, ex. `shellbase-1.5.5`:
+3. Make a release branch with Scala version and project version, ex. `shellbase-2.1.2`:
     ```
-    export RELEASE_VERSION=shellbase-1.5.5
+    export RELEASE_VERSION=shellbase-2.1.2
     git checkout -b ${RELEASE_VERSION}
     git add build.gradle
     git commit -m "[release] ${RELEASE_VERSION}"
     ```
 4. Perform a release in selected Scala versions:
     ```
-    ./gradlew build publish -PscalaVersion=2.11.12
-    ./gradlew build publish -PscalaVersion=2.12.15
-    ./gradlew build publish -PscalaVersion=2.13.8
+    ./gradlew shellbase-example:publish -PscalaVersion=2.11.12
+    ./gradlew shellbase-slack:publish -PscalaVersion=2.11.12
+    ./gradlew shellbase-core:publish -PscalaVersion=2.11.12
+    ./gradlew shellbase-example:publish -PscalaVersion=2.12.15
+    ./gradlew shellbase-slack:publish -PscalaVersion=2.12.15
+    ./gradlew shellbase-core:publish -PscalaVersion=2.12.15
+    ./gradlew shellbase-example:publish -PscalaVersion=2.13.8
+    ./gradlew shellbase-slack:publish -PscalaVersion=2.13.8
+    ./gradlew shellbase-core:publish -PscalaVersion=2.13.8
```

Comment:
That is to work around a strange problem when publishing to Sonatype. I have deterministically suffered from a problem of seeing 3 staging repositories created instead of one (even for a single Scala major version). None of these 3 staging repositories were complete and the (closing and) releasing could not proceed.